components/brave_ads/core/internal/history/ad_content_util_unittest.cc

```diff
@@ -40,6 +41,7 @@ TEST_F(BraveAdsAdContentUtilTest, Build) {
   expected_ad_content.creative_set_id = ad.creative_set_id;
   expected_ad_content.campaign_id = ad.campaign_id;
   expected_ad_content.advertiser_id = ad.advertiser_id;
+  expected_ad_content.segment = kSegment;
```

Comment:
This change highlighted a bug in `ad_content_info.cc` where we were not comparing segments